### PR TITLE
Add devops overview

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/devops-user-guide/understand-and-manage-devops-projects/overview.md [hunk 1]
```diff
@@ -6,3 +6,45 @@ linkTitle: "Overview"
...
+
+![sonarqube-result-detail](/images/docs/devops-user-guide/understand-and-manage-devops-projects/overview/sonarqube-result-detail.jpg)
+
+### KubeSphere CI/CD pipeline workflows
+
+A KubeSphere CI/CD pipeline runs on the back of the underlying Kubernetes Jenkins slaves. These Jenkins slaves can be dynamically scaled as they are dynamically provisioned or released based on the job status. The Jenkins master and slaves run as Pods on KubeSphere nodes. The master runs on one of the nodes with its configuration data stored in a volume. Slaves run across nodes while they may not be active all the time because they are created dynamically and deleted automatically as needed.
+
+When the Jenkins master receives a building request, it dynamically creates Jenkins slaves that run in Pods according to labels. At the same time, Jenkins slaves will be registered in the master. After slaves finish their jobs, they will be released and related Pods will be deleted as well.
+
+### Dynamically provision Jenkins slaves
```
Comment (shaowenchen) [inline content/en/docs/devops-user-guide/understand-and-manage-devops-projects/overview.md:42]: At present, the community uses agent instead of slave

Comment (Sherlock113) [reply content/en/docs/devops-user-guide/understand-and-manage-devops-projects/overview.md]: Fixed.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**